### PR TITLE
Implements #72 add priv_call support

### PR DESCRIPTION
**Pull request by magooster**
Added web3.priv.privacyCall that calls the JSON-RPC priv_call method

 Added example of using privacy call

Added integration test

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -289,6 +289,43 @@ function EEAClient(web3, chainId) {
...
+   * Invokes a private contract function locally
+   * @param options Options passed into `priv_call`
+   * options map can contain the following:
+   * privacyGroupId : Enclave id representing the receivers of the transactio
+   * to : Contract address,
+   * data : Encoded function call (signature + data)
+   * blockNumber: Blocknumber  efaults to "latest"
+   * @returns {Promise<AxiosResponse<T>>}
+   */
+  const privacyCall = options => {
```
Comment (akuanti) [inline src/index.js]: Can we call this `call` to be consistent with web3.js (`web3.eth.call`)?

[the remark this answers is not in the document]
Comment (akuanti) [reply src/index.js]: Or at least export it as `call` on line 338.

[the remark this answers is not in the document]
Comment (magooster) [reply src/index.js]: Updated as requested

**Comment by magooster** [comment]
> @magooster do you mind fixing the lint issues? `./node_modules/.bin/eslint --fix .` should do it. I'm not quite sure how the files got past the pre-commit linting stage. Thanks! Looks good otherwise.

Not sure what happened - have run the linter

**Pull request merged**